vimrc

```diff
@@ -79,7 +79,7 @@ augroup END
 
 autocmd FileType php setlocal tabstop=4 shiftwidth=4 softtabstop=4
 autocmd FileType python setlocal tabstop=4 shiftwidth=4 softtabstop=4
-autocmd FileType java setlocal tabstop=4 shiftwidth=4 softtabstop=4
+autocmd FileType java setlocal expandtab tabstop=2 shiftwidth=2 softtabstop=2
```

Comment:
This is the same as our default, right? Does this mean you delete this line entirely?

https://github.com/braintreeps/vim_dotfiles/blob/9826007dbdfac70728e55e4f0d76fe7b43754e25/vimrc#L38

Reply:
Ah good catch. I think you're right. I'll just send the line.

Reply:
6a5b6ff